CONTRIBUTING.rst

```diff
@@ -0,0 +1,333 @@
+.. raw:: html
+
+   <!-- SPDX-License-Identifier: MIT -->
+
+Contributing to disnake
+=======================
+
+-  `Bug Reports <#good-bug-reports>`__
+-  `Creating Pull Requests <#creating-a-pull-request>`__
```

Comment:
Can we hide this part in the built documentation? It's not needed there since the sidebar already lists all headings